### DISCUSSION TITLE
OS version for containers on Harbor (hub.opensciencegrid.org)

### DISCUSSION THREAD

**Discussion by aowen-uwmad** [PelicanPlatform/pelican#1185] [General]
Noticed that containers in https://hub.opensciencegrid.org/harbor/projects/883/repositories are using CentOS 8 Stream, which I believe is at EOL (or something; we are upgrading CHTC APs/EPs from 8 to 9).
Probably want to update the containers to use a newer OS as well.
Thoughts on having containers for different OS versions?

**Comment by bbockelm** [comment]
I think we should be able to fairly trivially re-base onto the OSG-provided versions of EL9.

@jhiemstrawisc - can you think of anything that would go wrong with this?  Should we just open a ticket for 7.9.0 and call it a day?

**Reply by jhiemstrawisc** [reply]
I don't foresee any issues with this, but we won't know until we try. I'll make a ticket so we can get this done in the next release.

**Comment by haoming29** [comment]
This issued is addressed in https://github.com/PelicanPlatform/pelican/issues/1199
